icons/aegisauthenticator.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Aegis Authenticator</title><path d="m16.89,20.44c.58.99-.17,2.29-1.32,2.28-.57,0-1.06-.31-1.32-.77h0s0,0,0,0c0,0,0-.02-.01-.02l-3.74-6.49c-.76-1-.02-2.49,1.24-2.47.61,0,1.13.35,1.39.86l3.77,6.64h0Zm-6.99-.04l-.37-.67c-1.04-1.76-3.51-.17-2.56,1.57l.34.64c1.08,1.74,3.51.26,2.59-1.53h0Zm.22-17.39L.23,20.38s-.05.09-.06.11c-.11.21-.17.45-.17.7,0,.84.69,1.52,1.53,1.51.53,0,1-.29,1.27-.71l7.95-13.95c.28-.46.81-.76,1.37-.7,0,0,.01,0,.02,0,.15.01.29.05.41.11.01,0,.06.03.03.01.03.01.06.03.08.05.05.03.1.06.15.09,0,0,.2.17.21.18.07.07.13.13.17.18,0,0,.1.14.16.24l7.81,13.76h0s.03.05.04.07c.27.42.74.7,1.27.7.84,0,1.52-.68,1.53-1.52,0-.26-.06-.5-.17-.71l-.02-.04h0s-7.72-13.59-7.72-13.59h0l-2.54-4.5-.08-.15h0c-.03-.05-.06-.11-.1-.17h0s0,0,0,0c-.22-.34-.61-.77-1.22-.81-.08-.01-.16,0-.23,0-.67.06-1.1.57-1.32.91"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Aegis Authenticator</title><path d="M11.92 1.258c-.67.06-1.1.57-1.32.91l-.48.86L.23 20.395l-.06.112a1.512 1.512 0 0 0 1.36 2.209c.53 0 1-.29 1.27-.71l7.95-13.95c.28-.46.81-.76 1.37-.7h.02c.15.01.29.05.41.11.01 0 .06.03.03.01.03.01.06.03.08.05l.15.09s.2.17.21.18c.07.07.13.13.17.18 0 0 .1.14.16.24l7.81 13.76s.03.05.04.07c.27.42.74.701 1.27.701.84 0 1.52-.681 1.53-1.521 0-.26-.06-.5-.17-.71l-.02-.04-7.72-13.59-2.54-4.5-.08-.15c-.03-.05-.06-.11-.1-.17-.22-.34-.61-.77-1.22-.81-.08-.01-.16 0-.23 0zm-.18 11.728c-1.26-.02-2 1.471-1.24 2.471l3.74 6.49c.01 0 .01.02.01.02.26.46.75.77 1.32.77 1.15.01 1.9-1.29 1.32-2.28l.01.03-3.77-6.64a1.56 1.56 0 0 0-1.39-.86zM8.283 18.98c-1.004.015-1.965 1.14-1.312 2.336l.34.641c1.08 1.74 3.51.26 2.59-1.53v-.009l-.372-.67c-.325-.55-.79-.774-1.246-.768z"/></svg>
```

Reply:
Thanks, it was better but needed slight tweaking for path positions.